packages/@react-aria/interactions/src/useMove.ts

```diff
@@ -12,12 +12,17 @@
 
 import {disableTextSelection, restoreTextSelection}  from './textSelection';
 import {MoveEvents, PointerType} from '@react-types/shared';
-import React, {HTMLAttributes, useMemo, useRef} from 'react';
+import React, {useMemo, useRef} from 'react';
 import {useGlobalListeners} from '@react-aria/utils';
 
 interface MoveResult {
   /** Props to spread on the target element. */
-  moveProps: HTMLAttributes<HTMLElement>
+  moveProps: {
```

Comment:
we'll want to keep this as HTMLAttributes<HTMLElement> so that we can add other listeners or attributes or remove some if we need to without it being a breaking change.